app/classifier/frame-annotator.jsx

```diff
@@ -186,38 +191,50 @@ export default class FrameAnnotator extends React.Component {
       }
     });
 
+    if (type === 'image'){
+      createdViewBox = `${this.props.viewBoxDimensions.x} ${this.props.viewBoxDimensions.y} ${this.props.viewBoxDimensions.width} ${this.props.viewBoxDimensions.height}`;
+      svgAnnotator = 
+        <svg ref="svgSubjectArea" className="subject" style={svgStyle} viewBox={createdViewBox} {...svgProps}>
+          <g ref="transformationContainer" transform={this.props.transform}>
+            <rect ref={(rect) => { this.sizeRect = rect; }} width={this.props.naturalWidth} height={this.props.naturalHeight} fill="rgba(0, 0, 0, 0.01)" fillOpacity="0.01" stroke="none" />
+            {type === 'image' && (
+              <Draggable onDrag={this.props.panByDrag} disabled={this.props.disabled}>
+                <SVGImage className={this.props.panEnabled ? 'pan-active' : ''} src={src} width={this.props.naturalWidth} height={this.props.naturalHeight} modification={this.props.modification} />
+              </Draggable>
+            )}
+
+            {(!!InsideSubject && !this.props.panEnabled) && (
+              <InsideSubject {...hookProps} />
+            )}
+
+            {Object.keys(tasks).map((taskName) => {
+              const PersistInsideSubject = tasks[taskName].PersistInsideSubject;
+              if (PersistInsideSubject) {
+                return <PersistInsideSubject key={taskName} {...hookProps} />;
+              }
+            })}
+          </g>
+        </svg>
+    }
+
+    if (type === 'text' && TaskComponent) {
+      const currentFrame = this.props.subject.locations[this.props.frame];
+      textAnnotator = <InsideSubject src={currentFrame['text/plain']} {...hookProps} />
```

Comment:
This should be something like 
```js
textAnnotator = InsideSubject ? <InsideSubject src={currentFrame['text/plain']} {...hookProps} /> : undefined;
```
since `InsideSubject` might not exist